cache/in-memory/src/updates.rs

```diff
@@ -292,6 +295,41 @@ impl UpdateCache for GuildUpdate {
     }
 }
 
+impl UpdateCache for IntegrationCreate {
+    fn update(&self, cache: &InMemoryCache) {
+        if !cache.wants(ResourceType::INTEGRATION) || self.guild_id.is_none() {
+            return;
+        }
+
+        super::upsert_guild_item(
+            &cache.0.integrations,
+            self.guild_id.unwrap(),
```

Comment:
The above should be an `if let` or something to extract the ID rather than do an unwrap